docs/MAIL-ETIQUETTE

```diff
@@ -104,26 +104,26 @@ MAIL ETIQUETTE
   messages"
 
   No matter what, we NEVER EVER respond to trolls or spammers on the list. If
-  you believe the list admin should do something particular, contact him/her
-  off-list. The subject will be taken care of as good as possible to prevent
-  repeated offenses, but responding on the list to such messages never lead to
+  you believe the list admin should do something in particular, contact him/her
+  off-list. The subject will be taken care of as much as possible to prevent
+  repeated offenses, but responding on the list to such messages never leads to
   anything good and only puts the light even more on the offender: which was
   the entire purpose of it getting sent to the list in the first place.
 
   Don't feed the trolls!
 
   1.7 How to unsubscribe
 
-  You unsubscribe the same way you subscribed in the first place. You go to
+  You can unsubscribe the same way you subscribed in the first place. You go to
   the page for the particular mailing list you're subscribed to and you enter
   your email address and password and press the unsubscribe button.
 
-  Also, this information is included in the headers of every mail that is sent
+  Also, unsubscribing is included in the headers of every mail that is sent
```

Comment:
Changed to: "Also, the instructions to unsubscribe are included in the headers of every 
  mail that is sent out to all curl related mailing lists and there's a footer 
  in each mail that links to the "admin" page on which you can unsubscribe and 
  change other options."